src/Data/Cache.hs

```diff
@@ -121,15 +121,15 @@ withAssocCache body = evalStateT (body assocCache) []
 -- testing
 -- testing
 -- [3,3,3,3,7,7]
--- 
+--
 -- >>> withAssocCache $ withFiniteCache (-1) $ testC
 -- one
 -- two
 -- testing
 -- [3,3,3,3,7,7]
 finiteCache :: Monad m => Int -> Cache m k a -> Cache (StateT Int m) k a
 finiteCache n c = Cache
```

Comment:
Also, I think this is one of those cases where point free syntax hinders readability instead of improving it, because it makes this line look different from the other ones.